js/user.js

```diff
@@ -1,25 +1,76 @@
-function main() {
-  var db = new Kinto({remote: "https://kinto.dev.mozaws.net/v1"});
-  var tasks = db.collection("users");
-
-  document.getElementById("form")
-    .addEventListener("submit", function(event) {
-      event.preventDefault();
-      tasks.create({
-        title: event.target.title.value,
-        url: event.target.url.value,
-        done: false
-      })
-      .then(function(res) {
-        event.target.title.value = "";
-        event.target.url.value = "";
-        event.target.title.focus();
-      })
-      .catch(function(err) {
-        console.error(err);
-      });
+var central_repository_server = "https://kinto.dev.mozaws.net/v1";
+function authenticate(){
```

Comment:
:+1: 

The ultimate goal is to be able to import factorize.js in an empty html page and being able to set and get the user storage URL.